lua/luasnip/nodes/snippet.lua

```diff
@@ -44,6 +44,13 @@ local function S(context, nodes, condition, ...)
 			return true
 		end
 	end
+
+	local nodes = nodes
+	if not nodes[1] then
+		-- Then it's a node-table rather than an array of nodes: auto-wrap it
+		nodes = { nodes }
+	end
```

Comment:
Very minor, but I think checking for `type(nodes) == "table"` is a bit more readable.

Reply:
That's the point, I can't. Both nodes *and* node lists are tables, so that doesn't help (hence the comment).

Reply:
Ohh yeah, you're right, mb!